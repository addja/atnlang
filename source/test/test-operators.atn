def main() {
	a = "hello ";
	b = "world";
	c = a + b + "!";
	print c + "%n";
	d = 3 % -2;
	print d+3, "%n";
	e = 3 <= 4;
	print e, "%n";
}
