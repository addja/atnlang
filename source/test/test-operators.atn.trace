main() <entry point>
return <line 9>
