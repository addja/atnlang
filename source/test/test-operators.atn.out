hello world!
4
true
